src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,37 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Pageblock Refresh
+Pageblocks have a refresh option to automatically renew the pageblock data without having to reload the entire page. 
+This is useful for pageblocks that display data that changes frequently, such as a list of recent records, an update to a chart, or a list of recent comments.
+
+=== Pageblocks With Refresh
+Not all pageblocks have the refresh option enabled. Below is a list of pageblocks that have the refresh option enabled.
+
+* Record List
+* Chart
+* Comment
+* Iframe
+* Metric
+* Progress
+* Record Organizer
+* Report
+* Social Feed
+* Calendar
```

Comment:
It's good you've pointed out what blocks support it.

From the looks of it, most blocks support auto-refresh; so what I'd consider is reworking the section to point out which blocks don't support it.
This would make it shorter & cleaner, which would also let me get rid of the section and incorporate it into the intro.

As an example:

```
== Pageblock Refresh

Pageblocks have a refresh option to ...
...

[IMPORTANT]
====
Geometry, Record, ..., and ... page blocks do not support auto-refresh.
====

...
...

=== Enabling Pageblock Refresh

...
```